packages/stryker-api/logging.ts

```diff
@@ -0,0 +1,4 @@
+export { default as LoggerFactory } from './src/logging/LoggerFactory';
+export { default as Logger } from './src/logging/Logger';
+export { default as LoggerFactoryMethod } from './src/logging/LoggerFactoryMethod';
+export { default as getLogger } from './src/logging/getLogger';
```

Comment:
Do we really need getLogger in a separate file or can we also just place the content here?

Reply:
It is how our entire API is build up right now. Everything in separate files. `getLogger` is also used from `LoggerFactoryMethod` and others. You want me to move the entire logging API to this file?

Reply:
Let's leave it like this for now.